src/updateTempResult.js

```diff
@@ -0,0 +1,26 @@
+import pressedAnotherOperator from "./pressedAnotherOperator";
+
+const updateTempResult = (result, key, hasLastChar, lastEquation) => {
```

Comment:
@grimxreaper Good job extracting this! Not sure about the name though. The word result doesn't conjure an image of the place that's updated since it can either have a result or just show the equation in process. Definitely don't dig "Temp" in the name since we likely only named it that because result was already taken. What would be a better name?

Also, is there a way we can simplify the parameter list / return value?

Reply:
How about 'updateDisplay'? 
Not sure how I would simplify the parameter list or what is returned...I'll keep thinking.

Reply:
@grimxreaper Great name suggestion! Looking forward to the simplification changes.

Reply:
@bryzaguy Is it okay if I make a new commit with the name changes? Or do you want me to go back in history and edit?

Reply:
Feel free to do a new commit. If this were a PR for an open source project we'd want to make sure our commits condensed. @grimxreaper